@xen-orchestra/lite/src/components/IntlMessage.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ElementType, ReactElement, ReactNode } from 'react'
+import { FormattedMessage, MessageDescriptor } from 'react-intl'
+import intlMessage from '../lang/en.json'
+
+// See https://formatjs.io/docs/react-intl/components/#formattedmessage
+interface Props extends MessageDescriptor {
```

Comment:
Why don't you extend `FormattedMessage` instead?

Reply:
`FormattedMessage` refers to a value, so it can't be used as a type.
By using `typeof FormattedMessage`, the property `$$typeof` become required as props